composer.json

```diff
@@ -41,6 +41,7 @@
         "blueimp/jquery-file-upload": "10.2.0",
         "carhartl/jquery-cookie": "1.*",
         "cboden/ratchet": "0.4.*",
+        "clue/multicast-react": "dev-master",
```

Comment:
HI. Could you specify a version constraint instead of using the master in here? That would be great

Reply:
it looks like you willed it into existence. they only just release 16 hours ago
https://github.com/clue/reactphp-multicast/releases/tag/v1.1.0